cobalt/media/base/drm_system.h

```diff
@@ -254,7 +254,7 @@ class DrmSystem : public base::RefCounted<DrmSystem> {
   TicketToSessionUpdateMap ticket_to_session_update_map_;
 
   MediaMetricsProvider media_metrics_provider_;
-  
+
```

Comment:
small nit, looks like this file doesn't need a change at all.

Reply:
I know the whitespace fix is unrelated but do you mind if I just keep that commit in the PR? The trailing whitespace hook was complaining about it on push, for some reason.